### PR TITLE
update for MAVLink for OpenTx setup page

### PR DESCRIPTION
**Pull request by olliw42**
title says it

### CODE REVIEW

File: docs/MTX.md [hunk 1]
```diff
@@ -15,23 +15,26 @@ Step 1. is beyond the scope of this article; please consult the project's discus
...
 ## mLRS Tx Module Setup
 
 - Tx Ch Source = mbridge
-- Tx Ser Dest = mbridge
-- Tx Snd RadioStat = off
 
-Note: There are situations in which it can be useful to enable "Tx Snd RadioStat", but you should do this only if you know what you are doing.
+Note: The parameter "Tx Snd RadioStat" should normally be set to off. There are situations in which it can be useful to enable "Tx Snd RadioStat", but you should do this only if you know what you are doing.
+
+Note: The parameter "Tx Ser Dest" can be set to serial, serial2 or mbridge. If serial or serial2 is selected, then a MAVLink router is enabled which routes the messages between serial or serial2, the mTX radio, and the mLRS link. This allows you to e.g. in addition connect a GCS to serial or serial2. If mbridge is selected, then a MAVLink router is not invoked and the MAVLink communication happens only between the mTX radio and the mLRS link.
```
Comment (jlpoltrack) [inline docs/MTX.md:30]: Slight edit:

'This allows you to connect a GCS to serial or serial2. If mbridge is selected, then the MAVLink router is not enabled and the MAVLink communication happens only between the mTX radio and the mLRS link.'

File: docs/MTX.md [hunk 2]
```diff
@@ -54,11 +57,11 @@ Note: 'x' refers to the serial port of your flight controller used for MAVLink
 
 ### Stream Rates
 
-In contrast to the other setups, the SRy parameters for the stream rates do not need to be configured here (i.e., can be left at zero). The MAVLink for OpenTx firmware requests the required streams itself.
+In contrast to other setups, the SRy parameters for the stream rates do not need to be configured here (i.e., can be left at zero). The mTX firmware requests the required streams itself.
```
Comment (jlpoltrack) [inline docs/MTX.md:60]: On CRSF page we recommend SERIAL_OPTIONS = 4096 | ignore stream rate change.  Should line be added here that this bit should not be set in the bitmask?

Comment (olliw42) [reply docs/MTX.md:60]: yeah ... I would wish ArduPilot would support more options here.

Comment (olliw42) [reply docs/MTX.md:60]: hm, thinking about it, I'm actually not sure how true the sentence is, i.e., if it indeed has absolutely no effect. Maybe it has some side effects. I guess one should say on can try ...

File: docs/MTX.md (hunk 1 above)
Comment (olliw42) [reply docs/MTX.md]: THX. done.

**Comment by jlpoltrack** [comment]
Looks good to me - will merge if you're all set.

**Comment by olliw42** [comment]
👍

**Comment by olliw42** [comment]
many THX!

**Pull request merged**